dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/ZonedInterval.java

```diff
@@ -70,6 +73,14 @@ public ZonedDateTime getEndInclusiveUTC() {
         return getEndInclusive().withZoneSameInstant(Z_UTC);
     }
 
+    @Nonnull
+    public ZonedInterval span(@Nullable ZonedInterval interval) {
```

Comment:
Please, avoid `@Nullable`, wherever possible. The argument should be expected to be non-null.